wallet/src/Dscp/Wallet/KeyStorage.hs

```diff
@@ -28,11 +29,17 @@ data StorageAccount = StorageAccount
 deriveJSON defaultOptions ''Storage
 deriveJSON defaultOptions ''StorageAccount
 
-storageFileName :: FilePath
-storageFileName = "run/tmp/wallet.json"
+getDataDir :: IO FilePath
+getDataDir = do
+    dir <- getXdgDirectory XdgData appName
+    createDirectoryIfMissing True dir
+    return dir
 
-storageLockFileName :: FilePath
-storageLockFileName = "run/tmp/wallet.json.lock"
+getStoragePath :: IO FilePath
+getStoragePath = getDataDir >>= return . (<> "/wallet.json")
```

Comment:
Can we use something cross-platform for concatenation?

Reply:
@goodsoft https://www.stackage.org/haddock/lts-9.11/filepath-1.4.1.1/System-FilePath-Posix.html#v:-60--47--62-

Reply:
:ok_hand:

Reply:
@volhovm @yegortimoshenko https://github.com/DisciplinaOU/disciplina/pull/167